ocaml/idl/markdown_backend.ml

```diff
@@ -154,8 +154,8 @@ let print_field_table_of_obj printer x =
   if x.contents=[] then
     printer ("Class "^(escape x.name)^" has no fields.")
   else begin
-    printer "|Field|Type|Qualifier|Description|";
-    printer "|:---|:---|:---|:---|";
+    printer "|Field            |Type               |Qualifier  |Description                         |";
```

Comment:
Yes; it seems tables in markdown are not that flexible as tables in html and allow very basic styling. I also tried converting all the documents to html first as an intermediate step, style them, and then make the pdf, but in that case I had trouble getting the metadata in cover.yaml into the pdf 😞